text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/s
 if(TARGET openvino_tokenizers)
     set(OPENVINO_TOKENIZERS_PATH $<TARGET_FILE:openvino_tokenizers>)
 else()
-    message(FATAL_ERROR "${TEST_TARGET_NAME} must be compiled as part of OpenVIINOGenAI to have the path to openvino_tokenizers hardcoded.")
+    set(OPENVINO_TOKENIZERS_PATH libopenvino_tokenizers.so)
```

Comment:
we could use https://cmake.org/cmake/help/latest/variable/CMAKE_SHARED_LIBRARY_SUFFIX.html and similar to prefix to create OS-independent name

Reply:
It'll go away after merging to GenAI anyway